rts/lra/README.md

```diff
@@ -0,0 +1,20 @@
+# MicroProfile LRA
+
+[MicroProfile LRA](https://github.com/eclipse/microprofile-lra) is a specification of an annotation based API that enables loosely coupled services to coordinate long running activities in such a way as to guarantee a globally consistent outcome without the need to take long duration locks on data.
+
+### MicroProfile Fault Tolerance
+Fault Tolerance provides developers with the following strategies for dealing with failure:
```

Comment:
We only use the `@Bulkhead` annotation in the code so we cannot use any of `@Timeout`, `@Retry`, `@CircuitBreaker` or `@Fallback` so we should not be documenting them

Reply:
Remark: we use asciidoc in the rest of the LRA module (./rts/lra/test/README.adoc ./rts/lra/API.adoc). What do you think about being consistent and changing the asciidoc files to markdown?

Reply:
Thanks @mmusgrov for the review. I updated the README, now it is an asciidoc file.